core/src/main/java/org/frankframework/jdbc/datasource/FrankResources.java

```diff
@@ -99,9 +101,9 @@ private DataSource loadDataSourceThroughDriver(Class<?> clazz, String url, Prope
 	}
 
 	private @Nullable Resource findResource(@Nonnull String name) {
-		List<String> parts = StringUtil.split(name, "/");
-		String prefix = parts.remove(0);
-		String resourceName = String.join("/", parts);
+		int slashPos = name.indexOf('/');
+		String prefix = name.substring(0, slashPos);
+		String resourceName = name.substring(slashPos + 1);
```

Comment:
This bit of code broke when I tried to make the result of `StringUtil.split()` unmodifiable.

In the end I reverted the change there, but decided to leave it in here because it should be slightly more optimal than doing a regex search and all the list manipulations.